content/pandas/dataframe/shape/shape.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: .shape # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: '.shape' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```